plugins/resource_monitor_plugin/resource_monitor_plugin.cpp

```diff
@@ -49,7 +49,12 @@ class resource_monitor_plugin_impl {
          ( "resource-monitor-interval-seconds", bpo::value<uint32_t>()->default_value(def_interval_in_secs),
            "Time in seconds between two consecutive checks of resource usage. Should be between 1 and 300" )
          ( "resource-monitor-space-threshold", bpo::value<uint32_t>()->default_value(def_space_threshold),
-           "Threshold in terms of percentage of used space vs total space. If used space is above (threshold - 5%), a warning is generated.  Unless resource-monitor-not-shutdown-on-threshold-exceeded is enabled, a graceful shutdown is initiated if used space is above the threshold. The value should be between 6 and 99" )
+           "Threshold in terms of percentage of used space vs total space. If used space is above (threshold - 5%), a warning is generated. "
+           "Unless resource-monitor-not-shutdown-on-threshold-exceeded is enabled, a graceful shutdown is initiated if used space is above the threshold. "
+           "The value should be between 6 and 99" )
+         ( "resource-monitor-space-absolute-gb", bpo::value<uint64_t>(),
```

Comment:
We check each file system monitored (the directories monitored might mount to different file systems). So the same absolute GB will apply to each file system. Probably we should call this out.